lib/cdo/activity_constants.rb

```diff
@@ -14,6 +14,14 @@ module ActivityConstants
   MANUAL_PASS_RESULT = 90
   BEST_PASS_RESULT = 100
 
+  PASSING_VALUES = [
+    MINIMUM_PASS_RESULT,
+    MAXIMUM_NONOPTIMAL_RESULT,
+    FREE_PLAY_RESULT,
+    MANUAL_PASS_RESULT,
+    BEST_PASS_RESULT
+  ]
```

Comment:
Agreed. Also this appears to only be used in the test to sample passing values. I'd be totally fine with defining this in the test for that purpose (i.e. a set of passing values to sample, but not necessarily the exhaustive set). It doesn't belong in activity_constants.